uwblueprint/jump-math
# fix: refactor common selects

## Notion ticket link
Work as part of [Add start / end time in test session](https://www.notion.so/uwblueprintexecs/Add-start-end-time-in-test-session-b09c7292d57c43a0a9fd271a82f5dd02)


## Implementation description
Medium-size refactor to selects within the app to use a common component. Rationale:
1. UI standardization: we currently have several different variants of `Select` components used throughout the app. This PR merges them all to one definition.
2. Code reuse: this moves a bunch of logic into a standard component, and simplifies access patterns (no more manual typing necessary). This is in preparation for the time picker, which uses selects.



## Steps to test
1. View the selects I updated (mainly: teacher creation, student sign-in, table filtering, multi-option assessment creation) and make sure they look good.



## What should reviewers focus on?
* 


## Checklist
